semcore/bulk-textarea/src/translations/en.json

```diff
@@ -0,0 +1,10 @@
+{
+  "BulkTextarea.ClearAllButton.buttonText": "Clear all",
+  "BulkTextarea.Counter.ofAllowedRows:sr-message": "of {rowsNumber} lines",
```

Comment:
I was thinking, since we count values not lines in the counter, shouldn't we use not "lines", but some other word here? Or maybe remove this word completely.